src/utilities/idf/WorkspaceObject_Impl.hpp

```diff
@@ -129,7 +129,7 @@ namespace detail {
      *  (non-extensible) fields and fields with empty data, if a default exists. */
     virtual boost::optional<std::string> getString(unsigned index, bool returnDefault = false, bool returnUninitializedEmpty = false) const override;
 
-    boost::optional<std::string> getField(unsigned index) const;
+    virtual boost::optional<std::string> getField(unsigned index, bool returnDefault = false) const override;
```

Comment:
I added a returnDefault param to match the others, with the original default of false